custom_components/waste_collection_schedule/waste_collection_schedule/source/bsr_de.py

```diff
@@ -1,130 +1,75 @@
 import datetime
-
 import requests
+from collections import namedtuple
 from waste_collection_schedule import Collection  # type: ignore[attr-defined]
```

Comment:
I think this can be removed
```suggestion
from waste_collection_schedule import Collection
```